migrations/sql/V2018.09.17.1344__MoreCountryMetadata.sql

```diff
@@ -0,0 +1,79 @@
+-- NOTE for review:
+--
+-- does it makes sense to prefix these tables with country_ or not?
+-- check null ness for everything
+-- everything that needs to be an enum table done
+-- no "data" in column names (e.g., thing_2008, thing_yf, etc)
+-- some of the not-null constraints need to be relaxed until we have the data import done - this needs writing out as a second import.
+
+
+CREATE TABLE country_gavi_region (
+  id TEXT NOT NULL,
+  name TEXT NOT NULL,
+  primary KEY (id)
+);
+COMMENT ON TABLE gavi_region
+  'include four types of gavi region interested by gavi doners';
+
+CREATE TABLE country_disease_endemic (
+  country TEXT  NOT NULL,
+  disease TEXT  NOT NULL,
+  FOREIGN KEY ("country") REFERENCES country (id),
+  FOREIGN KEY ("disease") REFERENCES disease (id));
+
+CREATE TABLE country_fragility (
+  id SERIAL,
+  country TEXT NOT NULL,
+  year INTEGER NOT NULL,
+  fragility text NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY ("country") REFERENCES country (id)
+);
+
+CREATE TABLE country_cofinance (
+  id SERIAL,
+  country TEXT NOT NULL,
+  year INTEGER NOT NULL,
+  cofinance text NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY ("country") REFERENCES country (id)
+);
+
+ALTER TABLE country_metadata
+  -- ADD COLUMN francophone INTEGER,
+  ADD COLUMN vxdel_segement TEXT NOT NULL,
```

Comment:
Copy out all these `NOT NULL` bits and put them into another sql file in a _second_ branch. Syntax is the same as here - within an `ALTER TABLE` block but use `ALTER COLUMN` rather than `ADD COLUMN`

Reply:
No problem.